content/concepts/discovery/mDNS.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "mDNS"
+description: "MDNS uses a multicast system of DNS records over a local network to enable peer discovery."
+weight: 224
+---
+
+## What is mDNS?
+
+mDNS, or multicast Domain Name System, is a way for nodes to use a multicast system
+of DNS records over a local network to discover and communicate with nodes. Nodes
+broadcast topics they're interested in instead of querying a central name server.
+The discovery, however, is limited to the peers in the local network. mDNS is commonly
+used on home networks to allow devices such as computers, printers, and smart TVs to
+find each other and connect. It uses a protocol called multicast to broadcast messages
+on the network, allowing devices to discover each other and exchange information.
+
+## mDNS in libp2p
+
+In libp2p, mDNS is used for service discovery.
+When a peer wants to connect to another peer in a libp2p network,
+it can use mDNS to resolve the hostname of the other peer to its multiaddr.
```

Comment:
No, we don't use it for address resolution. We use it to find _all_ other peers in the network.